src/Tribe/Status/Abstract_Commerce.php

```diff
@@ -0,0 +1,281 @@
+<?php
+
+
+/**
+ * Class Tribe__Tickets__Status__Abstract_Commerce
+ *
+ * @since TBD
+ *
+ */
+class Tribe__Tickets__Status__Abstract_Commerce {
+
+	/**
+	 * @var string a string for the completed order status, usually completed or publish
+	 */
+	public $completed_status_id;
+
+	/**
+	 * @var array an array of status names for a commerce
+	 */
+	public $status_names = array();
+
+	/**
+	 * @var array an array of status objects
+	 */
+	public $statuses = array();
+
+	/**
+	 * @var int the quantity of tickets sold for a post type
+	 */
+	protected $_qty = 0;
```

Comment:
We follow PSR-2 naming [conventions for object properties](https://www.php-fig.org/psr/psr-2/#42-properties) and should not prefix variables with `_`.